rules/good-email-subject/rule.md

```diff
@@ -7,6 +7,8 @@ authors:
     url: https://ssw.com.au/people/adam-cogan
   - title: Cameron Shaw
     url: https://ssw.com.au/people/cameron-shaw
+  - title: Levi Jackson
+    url: https://ssw.com.au/people/levi-jackson    
 related:
```

Comment:
Hey @Levijj22 

- [ ] 1. Please add an SEO description `seoDescription:` 

**Tip:** Use ChatGPT

Reply:
✅ Done - seoDescription: Learn the art of crafting impactful email subject lines to boost open rates, prioritize messages, and capture attention. This guide provides practical examples for writing clear, concise, and searchable subjects